docs/user/viewer-preferences.md

```diff
@@ -79,5 +79,5 @@ with open("output.pdf", "wb") as output_stream:
 ```
 
 The names beginning with a slash character are part of the PDF file format. They are
-included here to aid to anyone searching pypdf documentation
+included here to aid searching the pypdf documentation
```

Comment:
```suggestion
included here to ease to search the pypdf documentation
```

Reply:
included here to ease searching the pypdf documentation